src/providers/locations-service/locations-service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import ApiService from '../api-service/api-service';
+import { Location } from '../../models';
+import Print from '../../utils/print';
+
+/**
+ * Handles request on the BioPocket API that are related to the management of Locations of interest 
+ */
+@Injectable()
+export default class LocationsService {
+
+  private resourceName: string = '/locations'
+
+  constructor(private api: ApiService) {
+    this.api.version().subscribe(ver => Print.log(ver));
```

Comment:
This is not a good place to put this call. We might copy-paste it by mistake into other services. `AppComponent` is probably a better place.

Reply:
Yep, I forgot to remove it.
This was only used for the tests.
There's indeed no need for this to be there.